pkg/endpoint/bpf.go

```diff
@@ -127,7 +127,7 @@ func WriteIPCachePrefixes(fw *bufio.Writer, getBPFData getBPFDataCallback) {
 	}
 }
 
-func (e *Endpoint) writeHeaderfile(prefix string, owner Owner) error {
+func (e *Endpoint) writeHeaderfile(prefix string) error {
```

Comment:
nit: Removing owner is not part of the fix. I think I'll need it again soon for the templating stuff too, but I can always rebase and reinstate it.